src/listing.jsx

```diff
@@ -0,0 +1,226 @@
+import React, { Component } from 'react';
+import styled from 'styled-components/macro';
+import Octokit from '@octokit/rest';
+
+import theme from './theme';
+import mediaQueries from './media-queries';
+
+import { Container } from './components/container';
+import { Row } from './components/row';
+import { LoadingSpinner } from './components/loading-spinner';
+import { Button } from './components/button';
+import SmallArrow from './components/small-arrow';
+
+import IssueListing from './issue-listing';
+import SaveKey from './save-key';
+
+const ListingContainer = styled(Container)``;
+ListingContainer.displayName = 'ListingContainer';
+
+const RepoList = styled.ul``;
+RepoList.displayName = 'RepoList';
+
+const ColA = styled.div`
+    width: 100%;
+    padding-left: ${theme.gutter};
+    padding-right: ${theme.gutter};
+
+        @media (min-width: ${mediaQueries.min.medium}) {
+            width: 33.3333%;
+        }
+`;
+ColA.displayName = 'ColA';
+
+const ColB = styled.div`
+    width: 100%;
+
+        @media (min-width: ${mediaQueries.min.medium}) {
+            width: 66.6667%;
+        }
+`;
+ColB.displayName = 'ColB';
+
+const FullWidthCol = styled.div`
+    width: 100%;
+    padding-left: ${theme.gutter};
+    padding-right: ${theme.gutter};
+
+    &.slideup, &.slidedown {
+        @media (min-width: ${mediaQueries.min.medium}) {
+            max-height: 100%;
+        }
+    }
+
+    *[class*='listing__RepoToggle'] *[class*='button__Icon'] {
+        @media (min-width: ${mediaQueries.min.medium}) {
+            display: none;
+        }
+    }
+`;
+FullWidthCol.displayName = 'FullWidthCol';
+
+const RepoAccordion = styled(FullWidthCol)``;
+RepoAccordion.displayName = 'RepoAccordion';
+
+const RepoToggle = styled(Button)`
+    font-weight: normal;
+    font-size: ${theme.xlargeBaseFont};
+
+    *[class*='button__Text'] {
+        padding: 0;
+    }
+`;
+RepoToggle.displayName = 'RepoToggle';
+
+const SelectRepoButton = styled(Button)`
+    &.repo-selected {
+        text-decoration: underline;
+        color: ${theme.black};
+
+        *[class*='small-arrow__SmallArrowContainer'] {
+             border-bottom-color: ${theme.black};
+        }
+        
+        &:hover {
+            color: ${theme.primaryColor};
+
+            *[class*='small-arrow__SmallArrowContainer'] {
+                border-bottom-color: ${theme.primaryColor};
+            }
+        }
+    }
+`;
+SelectRepoButton.displayName = 'SelectRepoButton';
+
+export default class Listing extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            selectedRepo: null,
+            repos: '',
+            reposLoaded: false,
+            repoAccordionOpen: true
+        };
+
+        this.fetchRepos = this.fetchRepos.bind(this);
```

Comment:
Not sure if you are using arrow functions anywhere - but this just came to my mind! If using arrow functions, I'd use them everywhere, and if binding them, I'd do that everywhere for consistency! (This could already be ✅ if you are using bind everywhere 👍 )